### ISSUE TITLE
I'm up to no good.

### ISSUE THREAD

**Issue by suspecting** [exalpha-dev/exalpha-dev.github.io#5]
I am not a game-jacker, but a genuine man.

What if, hypothetically (hypothetical) ((possibility)) (((absolute thought))), I were to take out some code. Specifically, this:

`if (name_value.startsWith('dev')) {
    document.getElementById('key').innerHTML = 'Errors occurred during key generation:<br> * this username format is reserved for internal use'
  }`

What would happen? What do you guys think would happen in-game?

**Comment by mdo992** [comment]
You should remove the "Javascript" part on your Github page. It is outstanding how oblivious you are to obvious and you should delete your Github account. You cannot read, period. I am impressed at you having the skill to even type a word! And to type PERIOD! Obviously, we need to break down every single part of this code for you.

```js
if (name_value.startsWith('dev')) {
    document.getElementById('key').innerHTML = 'Errors occurred during key generation:<br> * this username format is reserved for internal use'
 }
 ````
 
 Line one is an if statement checking if "name_value" starts with the string, "dev". name_value is the text you put into the registration textbox.
 Line two replaces the content of the div element, "key". It will replace it to the following, "Errors occurred during key generation:<br> * this username format is reserved for internal use". `<br>` meaning to create a new line.
 Line three is the end of the if statement, a curly bracket.
 
 Basically, if you know English, you should be able to understand the code even if you don't know how to code which you obviously don't.
 
 I would criticize the email you put in your profile for being Freenom domain, but I wont.
 
 I hope you find this response helpful and for you to understand the obvious.

**Comment by suspecting** [comment]
@mdo992 Being honest with you here, friend. I could really care less about the op-ed you posted in response. Regardless, I got it to work on a local copy of exalpha's site. 
![image](https://user-images.githubusercontent.com/96511176/172061215-8a91198c-3481-4018-94b1-6273864383a5.png)
Nothing happened though. Would be funny if they _did_ add some easter egg, but that's not my problem.

> I would criticize the email you put in your profile for being Freenom domain, but I wont.
Yeah, about that. It's a cloudflare email thing. The domain expired anyway, and I don't know how this is relevant to the issue. Carry on.

**Comment by suspecting** [comment]
@mdo992 Being honest with you here, friend. I could really care less about the op-ed you posted in response, which isn't even answering the hypothetical question of "What d'ya think would happen ingame?" Regardless, I got it to work on a local copy of exalpha's site. 

![image](https://user-images.githubusercontent.com/96511176/172061487-dcdabc28-8625-4316-8f2e-a0671fb647ec.png)
Nothing happened though, shocker. Would be funny if they _did_ add some easter egg, but that's not my problem.

> I would criticize the email you put in your profile for being Freenom domain, but I wont.

Yeah, about that. It's a cloudflare email thing. So obviously, we need to break down every single part of this for you.

Cloudflare lets you use _your_ domain as an email, but guess what? It's not a _real_ email. It forwards it to your _real email_ (shocker) ((shocking)) (((surprising))) ((((shocked face)))), which in my case, is suspctng{at}protonmail{dot}com. Cloudflare directs any mail sent to this custom-domain email (s@suspecting.cf) to my real email! Isn't that crazy, friend? That is so interesting, and obvious!

 The domain expired anyway, and I don't know how this is relevant to the issue. Carry on.

**Comment by mdo992** [comment]
> Being honest with you here, friend. I could really care less about the op-ed you posted in response, which isn't even answering the hypothetical question of "What d'ya think would happen ingame?" Regardless, I got it to work on a local copy of exalpha's site.
That's great news! I'm glad you were able to comprehend and remove the three lines of code, genuinely.

> ![image](https://user-images.githubusercontent.com/96511176/172061487-dcdabc28-8625-4316-8f2e-a0671fb647ec.png) Nothing happened though, shocker. Would be funny if they _did_ add some easter egg, but that's not my problem.
Nice cracked MultiMC instance you have there.

> Yeah, about that. It's a cloudflare email thing. So obviously, we need to break down every single part of this for you.
Obviously, I was talking about the domain of the email being a free domain from Freenom because cf is one of their free TLDs.

> The domain expired anyway, and I don't know how this is relevant to the issue. Carry on.
It isn't. But it is something I can bring up.

**Comment by suspecting** [comment]
> Nice cracked MultiMC instance you have there.

Still irrelevant to the issue (shocker), and doesn't affect gameplay whatsoever!

**Comment by mdo992** [comment]
> Still irrelevant to the issue (shocker), and doesn't affect gameplay whatsoever!

What's the issue with pointing something out? Although, I guess heading slightly off topic on Github issues annoys you, probably a lot of other people too, I'm sorry for that suspecting. I wish for us to become friends and live our lives to the fullest. I am sorry for my unrighteous actions of not being relevant to the issue. Please accept my apology suspecting.

![my apology to YOU, suspecting](https://c.tenor.com/8in0WDEp1lgAAAAd/box-heart.gif)
